apps/designer/src/styles.css

```
@@ -0,0 +1,577 @@
+:root {
+  color: #171a2b;
+  background: #eef0f5;
+  font-family: "Avenir Next", "Segoe UI", system-ui, sans-serif;
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)